app/main.py

```diff
@@ -1 +1,45 @@
-#  write your code here
+class Animal:
+
+    def __init__(self, name: str, appetite: int, is_hungry: bool = True)\
```

Comment:
Place each argument in the `__init__` on a new line to improve the readability. There should be no arguments on the same line with opening and closing parenthesis. Follow this practice of writing __init__ in other classes as well as avoiding line breaking.

Reply:
Done